web/tests/libtest/env.py

```diff
@@ -257,15 +256,19 @@ def parts_to_url(codechecker_cfg):
 
 
 def get_workspace(test_id='test'):
-    tmp_dir = os.path.join(REPO_ROOT, 'build')
-    base_dir = os.path.join(tmp_dir, 'workspace')
-    if not os.path.exists(base_dir):
-        os.makedirs(base_dir)
+    """ return a temporary workspace for the tests """
```

Comment:
There is a similar function in the `analyzer` part of the CodeChecker. This has to be changed too:
https://github.com/Ericsson/codechecker/blob/54ce393b67dda9dd2463d59208252aa0265358d4/analyzer/tests/libtest/env.py#L35-L44